tests/cli/main.cpp

```diff
@@ -451,23 +508,23 @@ BOOST_FIXTURE_TEST_CASE( account_history_pagination, cli_fixture )
    {
       INVOKE(create_new_account);
 
-      // attempt to give jmjatlanta some peerplay
+       // attempt to give jmjatlanta some peerplay
       BOOST_TEST_MESSAGE("Transferring peerplay from Nathan to jmjatlanta");
       for(int i = 1; i <= 199; i++)
       {
          signed_transaction transfer_tx = con.wallet_api_ptr->transfer("nathan", "jmjatlanta", std::to_string(i),
                                                 "1.3.0", "Here are some CORE token for your new account", true);
       }
 
-      BOOST_CHECK(generate_block(app1));
+       BOOST_CHECK(generate_block(app1));
```

Comment:
Why this spaces were added? They make the code look bad: https://github.com/peerplays-network/peerplays/blob/6c354579ecc7fa638c08b53648ddc2d94ca78747/tests/cli/main.cpp#L519-L534
Please undo. Thx.